app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/AddPictureActivity.kt

```diff
@@ -0,0 +1,286 @@
+package com.github.HumanLearning2021.HumanLearningApp.view.dataset_editing
```

Comment:
this whole file seems duplicative. It should probably be removed.

Reply:
done.